build.xml

```diff
@@ -95,4 +96,19 @@
 			<arg path="tests"/>
 		</exec>
 	</target>
+
+	<target name="phpstan">
+		<exec
+			executable="vendor/bin/phpstan"
+			logoutput="true"
+			passthru="true"
+			checkreturn="true"
+		>
+			<arg value="analyse"/>
+			<arg value="-l"/>
```

Comment:
Options shortcuts are for humans. When written to bash scripts / build tools, the full option name should always be used for readability, i.e. `--level` in this case.

Reply:
@JanTvrdik Updated.